release_images.yml

```diff
@@ -90,13 +100,21 @@ release_images:
     framework: "tensorflow"
     version: "1.15.4"
     training:
-      device_types: ["gpu"]
-      python_versions: ["py37"]
+      device_types: ["cpu","gpu"]
+      python_versions: ["py36","py37"]
       os_version: "ubuntu18.04"
-      cuda_version: "cu110"
+      cuda_version: "cu100"
       example: False
       disable_sm_tag: True
```

Comment:
Please set `disable_sm_tag` to False, because CUDA 10.0 images for TF 1.15 are the "default".